docs/en/security-privacy-considerations.rst

```diff
@@ -0,0 +1,495 @@
+
+SPR-P-10
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon| 
+     - The protocol must ensure that no third party can read the Credential issued by the Issuer.
+
+To mitigate this threat, the IT Wallet specification requires the following security mechanisms in the issuance process:
+
+- *TLS*: Used in all communication between the Wallet and the Issuer, ensuring that data in transit is encrypted and protected from interception by attackers.
+- *Wallet Instance Attestation*: Ensures that the Wallet operates on a secure, trusted device and complies with the security standards required by the Issuer, providing additional assurance that the Issuer is interacting with a legitimate Wallet Instance.
+- *DPoP*: Ensures that the Holder of the access token possesses the private key associated with it, preventing attackers from reusing intercepted tokens.
+- *Holder binding*: Ties the Credential to a specific Holder, ensuring that only the legitimate Holder can use a Credential to authenticate with the Issuer.
+- *redirect_uri validation*: This validation ensures that the authorization response is sent to the correct and authorized endpoint, thereby preventing unauthorized interception by malicious actors. Ensuring the integrity of the **redirect_uri** is critical to avoid any manipulation or misdirection of the URI.  
+- *PKCE*:  Avoids injection of a legit authorization **code** in another session. 
+
+SPR-P-60
+~~~~~~~~
+.. list-table:: 
+   :widths: 8 92
+
+   * - |check-icon| 
+     - The protocol must ensure that during an interaction with a Verifier, an attacker cannot read PII.
+
+The IT Wallet complies with the requirements of P-60 by ensuring that all network connections are secured by TLS.
+
+In addition, as the Authorization Response contains PII is important that it is not sent to an attacker-controlled endpoint. 
+In the IT Wallet specification the **response_uri**, which is the endpoint where the Wallet sends the Authorization Response, is included in the signed Request Object, 
+which is verified by the Wallet using the Verifier's public key and trust chain. This guarantees that the Authorization Response is being sent to the correct endpoint. 
+Additionally, the Authorization Response is encrypted with the Verifier's public key, ensuring that only the intended recipient can decrypt and read the sensitive information, 
+further securing the transmission.
+
+Another endpoint to be validated is the **redirect_uri**, which is used to redirect the User back to the Verifier after the Credential presentation is complete. 
+In the IT Wallet specification, the **redirect_uri** is registered and validated beforehand during the Verifier onboarding using OpenID Federation. During the presentation 
+phase, the Wallet is able to validate this value by verifying the OpenID Federation Trust Chain related to the Verifier. 
```

Comment:
We need ref to the section describing how openid federation is used, therefore using the refs to the section about the trust infrastructure and removing all these explicit mentions to openid federation.

in the trust infrastructure section we use openid federation and we'll introduce all the required harmonization with x.509 during the milestone 0.9.0

for clarity, we can use example "such as using openid federation ... as defined ..."